third_party/webrender/webrender_api/src/lib.rs

```diff
@@ -92,6 +92,13 @@ impl Epoch {
     pub fn invalid() -> Epoch {
         Epoch(u32::MAX)
     }
+
+    /// The value of this [`Epoch`] as a u16 value. Note that if this Epoch's
+    /// value is more than u16::MAX, then the return value will be modulo
+    /// u16::MAX.
+    pub fn as_u16(&self) -> u16 {
```

Comment:
Does this require a patch file [here](https://github.com/servo/servo/tree/master/third_party/webrender/patches) ?

Reply:
Okay. I've uploaded a new version of this change that turns this into a trait in `gfx_traits`.